pkg/types/shellformat/shellformat.go

```diff
@@ -1,4 +1,4 @@
-package types
+package shellformat
```

Comment:
@fako1024 : this one had to move over to a sub-package. Otherwise the global-query server cannot be built on something other than Linux.